src/UCTNode.cpp

```diff
@@ -251,7 +251,7 @@ UCTNode* UCTNode::uct_select_child(int color, bool is_root) {
         }
     }
 
-    const auto numerator = std::sqrt(double(parentvisits));
+    const auto numerator = std::sqrt(double(parentvisits) * std::log(cfg_logpuct * parentvisits + cfg_logconst));
```

Comment:
Shouldn't this cast parent visits to double before doing the math, or do we not actually care because we expect only the bottom end to get lopped off. In any case it's inconsistent with the double cast in the sqrt() part.

Reply:
I added double cast in log for consistency, although I don't think it really matters.